src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -4905,20 +4972,41 @@ static PaError WriteStream( PaStream* s, const void *_buffer, unsigned long fram
 
         // Keep waiting again (on Vista it was noticed that WASAPI could SOMETIMES return NULL pointer
         // to buffer without returning AUDCLNT_E_BUFFER_TOO_LARGE instead)
-        if (wasapi_buffer == NULL)
+        if (deviceBuffer == NULL)
             continue;
 
+        // Get buffer of the Mono to Stereo mixer, otherwise use WASAPI device buffer directly
+        if (isMonoStereoConverter)
+        {
+            // Expand buffer if necessary (normally buffer is allocated by ActivateAudioClient())
```

Comment:
As for ReadStream: why?

Reply:
Would you please elaborate, I did not get what you mean.

Reply:
I meant, "why expand the buffer size". the comment should explain.